Cargo.toml

```diff
@@ -18,18 +18,21 @@ version = "0.2.0"
 features = ["full"]
 
 [dependencies]
-tokio = { version = "^1.35", features = ["signal", "sync", "time", "macros", "rt-multi-thread"], optional = true}
-tokio-util = {version = "0.7", optional = true}
-rdkafka =  { version = "0.36", features =  ["cmake-build"], optional = true}
-serde = { version = "1.0", features = ["derive"]}
-serde_json = {version = "1.0", optional = true}
-picky = { version =  "7.0.0-rc.8", default-features = false, features = ["x509"], optional = true}
-reqwest = { version = "0.12" , default-features = false, features = ["rustls-tls", "blocking"], optional = true}
-log =  "0.4"
+tokio = { version = "^1.35", features = ["signal", "sync", "time", "macros", "rt-multi-thread"], optional = true }
+tokio-util = { version = "0.7", optional = true }
+rdkafka = { version = "0.36", features = ["cmake-build"], optional = true }
+serde = { version = "1.0", features = ["derive"] }
+serde_json = { version = "1.0", optional = true }
+picky = { version = "7.0.0-rc.8", default-features = false, features = ["x509"], optional = true }
+reqwest = { version = "0.12", default-features = false, features = ["rustls-tls", "blocking"], optional = true }
+log = "0.4"
 thiserror = "1.0"
 prometheus = { version = "0.13", features = ["process"], optional = true }
-warp = { version = "^0.3.4", optional = true}
-lazy_static  = { version = "1.4", optional = true}
+lazy_static = { version = "1.4", optional = true }
+hyper = { version = "1.2.0", features = ["full"], optional = true }
```

Comment:
features = ["server"] should be sufficient

But add this to [dev-dependencies]
hyper = { version = "1.2.0", features = ["full"] }